### PR TITLE
Fill in more details in the outline

### LINKED ISSUE #1: Add more detail to the GitHub & Advanced Git Sections
Looking forward to those parts of class, but I'd really like more detail.

:hamburger: :star:

@muan Any suggestions

### PR DESCRIPTION
**Pull request by johndbritton**
This adds some more information about GitHub content and optional Advanced Git topics.

Fixes #1 

cc @muan I added CI just for you.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,7 +3,19 @@
...
+* pull
 
 # GitHub
+* forking
+* watching
+* issues
+* mentions
+* pull requests
+* auto-close
+* ci
```
Comment (johndbritton) [inline README.md]: @muan See, I told you we'd do it.

File: README.md [hunk 2]
```diff
@@ -3,7 +3,19 @@
...
 
 # Advanced Git (if we have time)
+* bisect
+* blame
+* rebase
```
Comment (muan) [inline README.md]: oh that reminded me, what about `reflog`?

Comment (johndbritton) [inline README.md]: Yeah, maybe we should cover that too.

File: README.md (hunk 1 above)
Comment (muan) [inline README.md]: :heart:

**Pull request merged**